db/sg_replicate_cfg.go

```diff
@@ -422,9 +422,9 @@ func (ar *ActiveReplicator) alignState(targetState string) error {
 
 }
 
-func (dbc *DatabaseContext) StartReplications() {
+func (dbc *DatabaseContext) StartReplications(ctx context.Context) {
 	if dbc.Options.SGReplicateOptions.Enabled {
-		base.DebugfCtx(dbc.SGReplicateMgr.loggingCtx, base.KeyReplicate, "Will start Inter-Sync Gateway Replications for database %q", dbc.Name)
+		base.DebugfCtx(ctx, base.KeyReplicate, "Will start Inter-Sync Gateway Replications for database %q", dbc.Name)
```

Comment:
Until we determine whether we're removing it, this should continue to use the SGReplicateMgr loggingCtx, as it's got custom CorrelationID configured.

Reply:
ok, will do.